syncpack.config.cjs

```diff
@@ -10,15 +10,11 @@ module.exports = {
 	workspace: false,
 
```

Comment:
Heads up that this was a breaking change in v9 https://github.com/JamieMason/syncpack/releases/tag/9.0.0